docs/configure/upgrading.md

```diff
@@ -22,6 +22,12 @@ To help ease the pain of keeping Storybook up-to-date, we provide a command-line
 
 <!-- prettier-ignore-end -->
 
+The `upgrade` command will use whichever version you specify. For example:
+
+- `npx storybook@latest upgrade` will upgrade to the latest version
+- `npx storybook@7.6.10 upgrade` will upgrade to `7.6.10`
+- `npx storybook@7 upgrade` will upgrade to the newest `7.x.x` version
+
 After running the command the script will:
 
 - Upgrade all Storybook packages in your project to the latest stable version
```

Comment:
This line needs to be corrected/removed. Not sure what a good wording would be though.